contracts/extensions/CompoundGovernorVotesUpgradeable.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.26;
+
+import {GovernorUpgradeable} from "@openzeppelin/contracts-upgradeable/governance/GovernorUpgradeable.sol";
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import {Time} from "@openzeppelin/contracts/utils/types/Time.sol";
+import {IComp} from "contracts/interfaces/IComp.sol";
+
+/// @title CompoundGovernorVotesUpgradeable
```

Comment:
Should we name this slightly differently to indicate that is has special handling for the COMP token, so it's not confused with the OZ one?  Perhaps `CompoundGovernorVotesCompUpgradeable` or something like that?

Reply:
that feels a bit duplicate with `Compound` at the start? Would it be more clear if we followed the OZ extension naming style for Compound specfic contracts and named it `GovernorVotesCompoundUpgradeable`?

Reply:
or `GovernorVotesCompUpgradeable`

Reply:
Renamed and inherited contract in https://github.com/ScopeLift/compound-governance-upgrade/pull/27/commits/a535b8c092cd466da4ecfb80011793c616ed11a0